app/utils/prepareAnswer.js

```diff
@@ -126,18 +126,23 @@ function prepareDailyStatsAnswer(body, n) {
 
 function prepareTestingResourceAnswer(stateResources) {
   if (stateResources.length === 0) {
-    return '\n\n\u{1F9EA} No COVID-19 Testing Labs found';
+    const temp = ['\n\n\u{1F9EA} No COVID-19 Testing Labs found'];
+    return temp;
   }
 
-  let data = `\n\n\u{1F9EA} COVID-19 Testing Labs in ${stateResources[0].state}, found ${stateResources.length} results\n`;
-
-  stateResources.forEach((stateResource) => {
-    data += `\n<b>${stateResource.nameoftheorganisation}, ${stateResource.city}\n`
-      + `</b>Description: ${stateResource.descriptionandorserviceprovided}.\n`
-      + `<i>Phone Number: ${stateResource.phonenumber}</i>\n`
-      + `<i>Website: ${stateResource.contact}</i>\n`;
-  });
+  const data = [`\n\n\u{1F9EA} COVID-19 Testing Labs in ${stateResources[0].state}, found ${stateResources.length} results\n`];
 
+  for (let i = 0; i < stateResources.length; i += 1) {
+    if (i === 0) {
+      data[i] += `\n<b>${stateResources[i].nameoftheorganisation}, ${stateResources[i].city}\n`
```

Comment:
This loop can still be minimized, you don't even need if statement. Simply create a string of data and use `push` to push it to the end of the array.

Reply:
Okay got it, will apply this change 😀